main.py

```diff
@@ -356,6 +362,53 @@ def __init__(self, *args):
 		self.actionExit.triggered.connect(self.onExitClicked)
 		self.actionSettings.triggered.connect(self.onSettingsClicked)
 
+		if not "search_for_updates" in self.settings.settings or not "buildId" in self.settings.settings:
+			self.settings.settings["search_for_updates"] = False
+			self.settings.settings["buildId"] = ""
+			self.settings.save()
+
+		if "search_for_updates" in self.settings.settings:
+			if self.settings.settings["search_for_updates"] == True:
+				if os.name == "nt":
+					projectJson = json.loads(requests.get("https://ci.appveyor.com/api/projects/mborgerson/xqemu-c5j6o/branch/master").text)
+					jobId = projectJson["build"]["jobs"][0]["jobId"]
+					print("Job ID: " + jobId)
+
+					if not jobId == self.settings.settings["buildId"]:
+						reply = QMessageBox.question(self, 'Update', "A new update is available, would you like to update?", QMessageBox.Yes, QMessageBox.No)
```

Comment:
You probably want to inform the user what the update is for: "XQEMU" (not "XQEMU-Manager").

Probably also want to inform them of the file-size, because in some countries, you still pay per data volume; we don't want angry users who have to pay $10 for clicking a button.